IOS_CREDENTIALS_EAS.md

```diff
@@ -5,20 +5,49 @@ To distribute your iOS application, you must set up all the credentials required
+| Push Notification capability only |                                                                       |
+|-----------------------------------|-----------------------------------------------------------------------|
+| Local credentials                 | Works but unnecessary to go through extra effort to use local signing |
+| Managed credentials               | Works (simple setup)                                                                 |
+
+| Multiple capabilities |                                  |
+|-----------------------|----------------------------------|
+| Local credentials     | Works but requires complex setup |
+| Managed credentials   | Works unreliably                 |
+
+If you are unsure, run `expo prebuild` and look at the native entitlements file of your main target (in `ios` directory). Other config plugins may be adding capabilities here. Or check the full [capabilities](https://docs.expo.dev/build-reference/ios-capabilities/#supported-capabilities) list supported by Expo.
+
+
+#### Simple setup explanation
+Since there is only one capability (Push Notifications) required by your app, everything should just work. OneSignal will automatically add the capability to your app along with the necessary App Groups and EAS should sync them with your Apple Developer account. You can choose to use managed signing on EAS without a problem. This works because OneSignal will add the capability to both targets' entitlements files.
+
+#### Complex setup explanation
+Having additional capabilities on your app will result in a more complex setup, including having to use local credentials. The following is an explanation of why. It is not imperative to understand but we have included it for context.
+
+The OneSignal plugin will create an additional target: `OneSignalNotificationServiceExtension`. This target is the iOS service extension that adds support for Confirmed Deliveries, badges, media attachments, action buttons, and influenced opens (Firebase Analytics). Learn more [here](https://documentation.onesignal.com/docs/service-extensions). The plugin will automatically add app groups to both targets' entitlements files which are required if you want to make use of these features. They allow your app to execute code when a notification is received, even if your app is not active.
+
+Each target has an entitlement file. If there are multiple entitlements files per project, EAS will randomly pick one and apply it to both targets. To mitigate this, we have added the push capability entilement to both files (even though it's really only necessary in the main target). This can still pose a problem if it picks the NSE's entitlements file since it *could* break any other capabilities in your project, given that file will not have capabilities added to your main target by yourself or other plugins.
```

Comment:
this is not a problem if you are using `EXPO_NO_CAPABILITY_SYNC` or crednetials.json